src/In.ProjectEKA.TMHHip/DataFlow/Collect.cs

```diff
@@ -0,0 +1,534 @@
+            }
+
+            if (!list.Any()) return Option.None<MedicationResponse>();
+            var medicationResponse = new MedicationResponse
+            {
+                Entry = list,
+                Id = Uuid.Generate().Value.Split(":").Last(),
+                ResourceType = "Bundle",
+                Type = "collection"
+            };
+            return Option.Some(medicationResponse);
+        }
+
+        private static async Task<Option<ObservationResponse>> FetchClinicalNotes(DataRequest dataRequest,
+            List<ClinicalNote> clinicalNotes, string patientName)
+        {
+            LogDataRequest(dataRequest);
+            var uuid = Uuid.Generate().Value;
+            var id = uuid.Split(":").Last();
+            var observations = (from clinicalNote in clinicalNotes
+                where WithinRange(dataRequest.DateRange, clinicalNote.CreatedDate)
+                select new ObservationRepresentation
+                {
+                    FullUrl = uuid,
+                    Resource = new Resource(HiType.Observation, id, "final", new Code("Clinical notes"),
+                        new Subject(patientName), new List<Performer> {new Performer(clinicalNote.UserName)},
+                        clinicalNote.CreatedDate, clinicalNote.Note),
+                }).ToList();
+
+            if (!observations.Any()) return Option.None<ObservationResponse>();
+            var observationResponse = new ObservationResponse
+            {
+                Entry = observations,
+                Id = Uuid.Generate().Value.Split(":").Last(),
+                ResourceType = "Bundle",
+                Type = "collection"
+            };
+            return Option.Some(observationResponse);
+        }
+
+        private static async Task<Option<ConditionResponse>> FetchSwellingSymptomData(DataRequest dataRequest,
+            List<SwellingSymptomData> swellingSymptomsData, string patientName)
+        {
+            LogDataRequest(dataRequest);
+            var uuid = Uuid.Generate().Value;
+            var id = uuid.Split(":").Last();
+            var conditions = (from swellingSymptomData in swellingSymptomsData
+                where WithinRange(dataRequest.DateRange, swellingSymptomData.RecordedDate)
+                select new ConditionRepresentation
+                {
+                    FullUrl = uuid,
+                    Resource = new ConditionResource(HiType.Condition, id, new Code("Symptoms of Swelling"),
+                        new Subject(patientName), swellingSymptomData.RecordedDate,
+                        new List<Note>
+                        {
+                            new Note(swellingSymptomData.SwellingSite), new Note(swellingSymptomData.SwellingLtrl),
+                            new Note(swellingSymptomData.SwellingSize)
+                        }),
+                }).Cast<ICondition>().ToList();
+
+            if (!conditions.Any()) return Option.None<ConditionResponse>();
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4940156)